From: suzuki toshiya <mpsuzuki@hiroshima-u.ac.jp>
Date: Mon, 8 Apr 2019 02:56:50 +0000
Subject: [PATCH] Makefile.sources: move font-variations.c (a test code for
 Variation Font with FreeType2 functions) from test_sources to
 ft_font_test_sources, to restrict the test to the case with FreeType2
 backend. See discussion in
 https://lists.cairographics.org/archives/cairo/2019-April/028858.html

---
 test/Makefile.sources | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/Makefile.sources b/test/Makefile.sources
index c47131faffe1..cc3d81f7e0be 100644
--- a/test/Makefile.sources
+++ b/test/Makefile.sources
@@ -144,7 +144,6 @@ test_sources = \
 	font-face-get-type.c				\
 	font-matrix-translation.c			\
 	font-options.c					\
-	font-variations.c				\
 	glyph-cache-pressure.c				\
 	get-and-set.c					\
 	get-clip.c					\
@@ -399,6 +398,7 @@ pthread_test_sources =					\
 
 ft_font_test_sources = \
 	bitmap-font.c \
+	font-variations.c \
 	ft-font-create-for-ft-face.c \
 	ft-show-glyphs-positioning.c \
 	ft-show-glyphs-table.c \
